CONTRIBUTING.md

```diff
@@ -13,7 +13,7 @@ If your repo has certain guidelines for contribution, put them here ahead of the
 -->
 
 - [Contributor License Agreement](https://git.k8s.io/community/CLA.md) Kubernetes projects require that you sign a Contributor License Agreement (CLA) before we can accept your pull requests
-- [Kubernetes Contributor Guide](https://git.k8s.io/community/contributors/guide) - Main contributor documentation, or you can just jump directly to the [contributing section](https://git.k8s.io/community/contributors/guide#contributing)
+- [Kubernetes Contributor Guide](https://git.k8s.io/community/contributors/guide) - Main contributor documentation, or you can just jump directly to the [contributing page](https://git.k8s.io/community/contributors/guide/contributing.md)
```

Comment:
```suggestion
- [Kubernetes Contributor Guide](https://k8s.dev/guide) - Main contributor documentation, or you can just jump directly to the [contributing page](https://www.kubernetes.dev/docs/guide/contributing/)
```

these days we're trying to direct folks to the k8s.dev site instead of directly to github for the docs.

Reply:
I would be happy to change this. But wondering if are there any references about which link should we use?

Last month we have different opinion on it [at this k/website PR](https://github.com/kubernetes/website/pull/35648#discussion_r936208312).

CC @reylejano @natalisucks

Reply:
In general at this point we want to direct people to the website. Its much more searchable than github and will also have a new contributor course.